safe/definitions/hazard_classifications.py

```diff
@@ -657,6 +657,90 @@
     'classification_unit': tr('hazard zone')
 }
 
+inundation_classes = {
+    'key': 'inundation_classes',
+    'name': tr('Inundation classes (Dam Failure)'),
```

Comment:
If the classification is specific to dam break, maybe you should rename the variable `inundation_dam_class` L660 and key also. You can rename the name to 'Inundation classes', this class will be displayed only for the dam failure hazard already so not need to specify it again in the name here.